src/store/audio.ts

```diff
@@ -1220,12 +1220,16 @@ export const audioStore = createPartialStore<AudioStoreTypes>({
       if (style == undefined) throw new Error("assert style != undefined");
 
       const styleName = style.styleName || DEFAULT_STYLE_NAME;
+
+      const projectName = getters.PROJECT_NAME ?? "";
+
```

Comment:
（細かいですが）

ここは意味がまとまってるので空行はない方がいいかもです！
まあ人によりそうですが･･･！
```suggestion

      const projectName = getters.PROJECT_NAME ?? "";

```